pkg/virt-handler/isolation/validation.go

```diff
@@ -14,19 +15,22 @@ const (
 	QEMUIMGPath = "/usr/bin/qemu-img"
 )
 
-func GetImageInfo(imagePath string, context IsolationResult) (*containerdisk.DiskInfo, error) {
+func GetImageInfo(imagePath string, context IsolationResult, config *v1.DiskVerification) (*containerdisk.DiskInfo, error) {
+	memoryLimit := fmt.Sprintf("%d", config.MemoryLimit.Value())
+
 	// #nosec g204 no risk to use MountNamespace()  argument as it returns a fixed string of "/proc/<pid>/ns/mnt"
 	out, err := virt_chroot.ExecChroot(
-		"--user", "qemu", "--memory", "1200", "--cpu", "10", "--mount", context.MountNamespace(), "exec", "--",
+		"--user", "qemu", "--memory", memoryLimit, "--cpu", "10", "--mount", context.MountNamespace(), "exec", "--",
 		QEMUIMGPath, "info", imagePath, "--output", "json",
 	).Output()
 	if err != nil {
+		memError := fmt.Errorf("trying to increase the verification memory usage limit by specifying the DiskVerification field in the developer configuration options in the KubeVirt CR might solve this")
```

Comment:
The error hangs a little bit in the air, since if we reject the disk for valid reasons (like having backing files), we would still print this error.

Reply:
Wouldn't that happen in containerdisk.VerifyImage() though? the error messages this is 'augmenting' are prefixed with 'failed to invoke qemu-img' which sounds like an internal error occurred during verification rather than verification did not succeed.

I don't see where in this function we actually reject something because of it not being valid, qemu-img succeeds (meaning the exit code is 0) with all kinds of files.

Reply:
Right. :+1: 

Still I feel slightly uneasy about this. We may suggest people to increase the limit until a disk which should not pass the validation (due to being malicious) finally passes it.

Reply:
I see your point, @iholder-redhat maybe we want to hold this error message and talk about it a bit more and include it in a following PR if we think it makes sense?

Reply:
Is it possible to have indication about the failure reasons? It would be best if we printed this message only if we failed on memory issues.

Reply:
Yes, please remove the error message for now or formulate it differently.

Reply:
Oh sorry, my page wasn't refreshed, I didn't see all of the follow-up comments.

Makes sense to me to include this in a follow-up PR, but I do think it's very important. As you said, this is a `very niche and obscure setting that likely 99.99% of admins won't even know about`, so without indication from us no one will think to change this setting if obscure `cannot allocate memory` messages pop up.

Reply:
I've removed the error message for now, please re-review, thanks!